docs/installation.rst

```diff
@@ -81,19 +94,24 @@ Viewing the local server
 
   all the request sent to local.treeherder.mozilla.org will be proxied to it by varnish/apache.
 
-
-* For development you can use the django runserver instead of gunicorn:
+* Or for development you can use the django runserver instead of gunicorn:
 
   .. code-block:: bash
 
-     (venv)vagrant@precise32:~/treeherder-service$ python manage.py runserver
+     (venv)vagrant@precise32:~/treeherder-service$ ./manage.py runserver
+
+  this is more convenient because it automatically refreshes every time there's a change in the code. However it can consume too much memory when under load, causing the OS to kill it.
```

Comment:
We may add, explaining "under load" by saying something to the effect of (such as data ingestion or log parsing locally)